modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -221,8 +230,17 @@ public void validate(
 					getObjectValidationRuleEngine(
 						objectValidationRule.getEngine());
 
-			Map<String, Object> results = objectValidationRuleEngine.execute(
-				variables, objectValidationRule.getScript());
```

Comment:
@p-albuquerque since `results` is only used inside the `for`, you can keep it here by doing: `Map<String, Object> results = new HashMap<>()`